doc/content/verification_and_validation/val-2c.md

```diff
@@ -114,34 +113,117 @@ The case and model parameters used in both approaches in TMAP8 are listed in [va
-
 !media comparison_val-2c.py
        image_name=val-2c_comparison_TMAP8_Exp_T2_Ci.png
        style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
        id=val-2c_comparison_T2
-       caption=Comparison of TMAP8 calculations against the experimental data for T$_2$ concentration in the enclosure over time. TMAP8 matches the experimental data well, with an improvement when T$_2$ is injected over a given period rather than immediately.
+       caption=Comparison of TMAP8 calculations against the experimental data for T$_2$ concentration in the enclosure over time. TMAP8 matches the experimental data well, with an improvement when T$_2$ is injected over a given period rather than immediately. Calibration of the delayed injection model delivers further improvements.
 
 !media comparison_val-2c.py
        image_name=val-2c_comparison_TMAP8_Exp_HTO_Ci.png
        style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
        id=val-2c_comparison_HTO
-       caption=Comparison of TMAP8 calculations against the experimental data for HTO concentration in the enclosure over time. TMAP8 matches the experimental data well, with an improvement when T$_2$ is injected over a given period rather than immediately.
+       caption=Comparison of TMAP8 calculations against the experimental data for HTO concentration in the enclosure over time. TMAP8 matches the experimental data well, with an improvement when T$_2$ is injected over a given period rather than immediately. Calibration of the delayed injection model delivers further improvements.
+
+As shown in the red curve in [val-2c_comparison_T2] and [val-2c_comparison_HTO], using [MOOSE's stochastic tools module](https://mooseframework.inl.gov/modules/stochastic_tools/index.html) notably increased the agreement between the modeling predictions and experimental data for both the T$_2$ and HTO concentrations.
+The RMSPE for T$_2$ decreases from 58.5 % to 35.01 % and the RMSPE for HTO decreases from 74.77 % to 66.39 %.
+Note that although the calibration approach is similar to the one presented in [!cite](Simon2025), the results presented here include more simulations and the quality of the calibration is increased here (RMSPE values are further decreased here).
+
+[val-2c_calibration_input] and [val-2c_calibration_output] show the evolution of the model parameter values and of the optimization metric (time integral of $g$ defined in [eq:optimization_metric]) as a function of the number of simulation. The calibrated model corresponds to the highest value.
+
+!media comparison_val-2c.py
+       image_name=val-2c_pss_inputs.png
+       style=width:60%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=val-2c_calibration_input
+       caption=Evolution of the model parameter values as a function of the number of simulation.
```

Comment:
"Number of simulations" or "Number of steps"? There is a mismatch in this caption compared to the axes of the plots above.

Reply:
I changed it all to number of simulations.